### DISCUSSION TITLE
Woher kommen die Daten?

### DISCUSSION THREAD

**Discussion by manfred-gentsch** [Stellarium/stellarium#3871] [Q&A]
Hallo. 

Mich interessiert, wie und wo die Entfernungsdaten Erde - Erdenmond ermittelt werden.

Gruß
Manfred

**Comment by gzotti** [comment]
Section 5.5.3 and Appendix F and the rest of the User Guide contain valuable information for those who want to know more. For even deeper insight you must look into the code and comments therein.

**Comment by manfred-gentsch** [comment]
Vielen Dank für die schnelle Antwort.
Eigentlich wollte ich nur wissen von welcher LLR-Bodenstation (Observatorium) gemessen wird und wo auf dem Mond der Reflektor steht.
Gelten die Kilometerangaben vom Erdinneren bis zum Mondinneren? Oder wird von der Erdoberfläche zur Mondoberfläche gemessen? Bitte entschuldigen Sie meine laienhafte Anfrage.

**Reply by gzotti** [reply]
If you use topocentric corrections, distance is most likely topocentric observer position to center of Moon, as given by the respective model. If your observer is in the geocenter, the distance given is geocenter-selenocenter. 

Stellarium is not connected to laser range instruments or other live instrumentation apart from your system clock and optional geolocation services. Our values are not "measurements", but "model predictions" which hopefully agree with certified "model predictions", at least sufficiently well for the purpose of your application.

**Comment by manfred-gentsch** [comment]
Vielen Dank für die Erläuterung. Somit ist meine Frage beantwortet und ich habe einen Blick in den Hintergrund des Stellarium Planetariums machen können. Darf ich Ihre Aussage so weitergeben?

**Reply by gzotti** [reply]
Who are you reporting to? Which purpose will your report fulfill, in which context?

**Comment by manfred-gentsch** [comment]
Hallo Herr Zotti,
es bestehen keinerlei Ambitionen Ihre Aussage wissenschaftlich oder literarisch zu veröffentlichen. Sie soll nur dazu dienen meinen Freunden im Forum zu erklären wie die Entfernungsangaben Erde - Mond in Stellarium generiert werden, Gerade nicht durch Messungen, sondern durch "Modellvorhersagen". Ich glaube, einigen Freunden ist das so nicht bekannt. Kann ich irgendwo nachlesen, wie "Modellvorhersagen" gemacht werden? Allerdings bin ich in keiner Weise vorgebildet.

Vielen Dank für Ihre Mühen!

**Reply by gzotti** [reply]
This topic fills whole textbooks. See User Guide, Bibliography, entries authored by J. Meeus, for a start.

**Comment by manfred-gentsch** [comment]
Nochmals Danke. Ich schließe den Kommentar.
